frontend/public/reducers/features.ts

```diff
@@ -90,7 +93,10 @@ export const stateToFlagsObject = (state: RootState): FlagsObject =>
   state[featureReducerName].toObject();
 
 export const stateToProps = (desiredFlags: string[], state: RootState): WithFlagsProps => ({
-  flags: _.pick(stateToFlagsObject(state), desiredFlags),
```

Comment:
@vojtechszocs FIY `_.pick` behavior is different to original implementation. If we request a flag which does not exist in `state`, then resulting `flags` object wouldnt contain the key at all but with `reduce` it does

Reply:
Got it, thanks for clarification, missed this detail before.